docs/UserGuide.md

```diff
@@ -24,7 +24,9 @@ This user guide provides you with the necessary information on how to become an
 You can familiarize yourself with the terminologies, syntax and icons used in this user guide by reading the following sub-sections.  
 
 ### GUI terminologies
-The figure below shows the GUI of ProductiveNUS, with its components labelled. 
+The figure below shows the GUI of ProductiveNUS, with its sections labelled. 
+![GUI](images/GUI.png)
```

Comment:
I think there is a black outline for this photo. Minor nitpick but could perhaps crop it out :)

Reply:
I noticed the other images are like this as well. Minor issue though, we can fix it in the future!

Reply:
Fixed!